tests/pds/pds_test.go

```diff
@@ -0,0 +1,78 @@
+package tests
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/portworx/torpedo/tests"
+
+	"github.com/portworx/torpedo/pkg/pds"
+)
+
+func TestPDS(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Torpedo: PDS")
+}
+
+var (
+	controlPlane  *pds.ControlPlane
+	targetCluster *pds.TargetCluster
+)
+
+var _ = BeforeSuite(func() {
+	cpKubeconfig, err := tests.GetSourceClusterConfigPath()
+	Expect(err).NotTo(HaveOccurred())
+	Expect(cpKubeconfig).To(BeAnExistingFile())
+	controlPlane, err = pds.NewControlPlane(cpKubeconfig)
+	Expect(err).NotTo(HaveOccurred())
+
+	targetKubeconfig, err := tests.GetDestinationClusterConfigPath()
```

Comment:
We're gonna have to think about how we're going to be talking to both clusters when each of them is likely to have completely separate auth.